eclipse-collections/src/main/resources/META-INF/services/org.eclipse.collections.api.factory.set.MutableSetFactory

```diff
@@ -0,0 +1 @@
+org.eclipse.collections.impl.set.mutable.MutableSetFactoryImpl
```

Comment:
I forgot to highlight this earlier. I removed the usage of auto-service and just moved these META-INF/services files into src/main/resources. Not too difficult.